release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -152,9 +152,162 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://githu
+```ballerina
+task:Error? output = task:configureWorkerPool(6, 7000);
+```
+
+- Schedules the Ballerina job at a specified time.
+```ballerina
+import ballerina/task;
+import ballerina/time;
+
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+
+time:ZoneOffset zoneOffset = {hours: 5, minutes: 30};
+time:Civil time = {
+    year: 2021,
+    month: 4,
+    day: 12,
+    hour: 23,
+    minute: 20,
+    second: 50.52,
+    timeAbbrev: "Asia/Colombo",
+    utcOffset: zoneOffset
+};
+task:Error|task:JobId id = task:scheduleOneTimeJob(new MyJob(), time);
+```
+
+- Schedules the recurring Ballerina job according to the given duration.
+```ballerina
+import ballerina/task;
+
+class MyJob {
+   *task:Job;
+
+   public function execute() {
+       // logic goes here
+   }
+}
+task:Error|task:JobId id = task:scheduleJobRecurByFrequency(new MyJob(), 1);
+```
+
+- Unschedules the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:unscheduleJob(id);
+```
+
+- Pauses all the jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:pauseAllJobs();
+```
+
+- Resumes all the jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:resumeAllJobs();
+```
+
+- Pauses the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:pauseJob(id);
+```
+
+- Resumes the particular job.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:resumeJob(id);
+```
+
+- Gets all the running jobs.
+```ballerina
+import ballerina/task;
+
+task:Error? result = task:getRunningJobs();
+```
+
+#### Time Package Updates
+
+Revamped the entire time package as follows:
+
+- Introduced the `time:Utc` record to represent the UTC timestamp.
+- Introduced `time:Civil` to represent the localized time.
+- Added necessary APIs to do time generation, manipulations, and conversions.
+
+Steps for migration from the previous version to the current version are listed [here](https://github.com/ballerina-platform/ballerina-standard-library/issues/1079).
+
+#### Cache Package Updates
+
+- Introduced the new configuration as `EvictionPolicy` to set the eviction policy in the `CacheConfig` record.
+
+The `EvictionPolicy` record has been introduced with the option `LRU` as the module only supports the LRU eviction policy to evict the cache data when the cache is full.
+
+- Removed the `AbstractEvictionPolicy` object.
+
+This object had the common APIs for the cache eviction functionalities to implement the custom eviction policy. It has been removed by introducing the above configuration.
+
+#### New `xmldata` Package
+
+A new module is added to convert a natural representation of data in XML into a natural representation of data in JSON and vice-versa.
+
+- Converts a JSON object to an XML representation.
+```ballerina
+json data = {
+    name: "John",
+    age: 30
+};
+xml|Error x = fromJson(data);
```

Comment:
Needs to be a qualified identifier, right? With the import? Please check the other example too.